util/src/main/java/io/kubernetes/client/util/Config.java

```diff
@@ -26,6 +26,8 @@
 
 import javax.net.ssl.KeyManager;
 
+import org.apache.commons.codec.binary.Base64;
```

Comment:
I can't find a good one. Eclipse will do it, but then I'd have to use eclipse :)